recipes/torchhd/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "torch-hd" %}
+{% set version = "2.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4ffb7c627441aa71a3e286cab035bbe9c52d7ccfaf2605c613a22cdfea1eeccb
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy
+    - pandas
+    - python
+    - requests
+    - pytorch
+
+test:
+  imports:
+    - torchhd
+    - torchhd.datasets
```

```suggestion
test:
  imports:
    - torchhd
    - torchhd.datasets
  commands:
    - pip check
  requires:
    - pip
```
This just double checks the dependencies.